python/tests/base.py

```diff
@@ -203,6 +203,5 @@ def assertDataFramesEqual(self, dfA, dfB):
         sortedB = dfB.select(colOrder)
         # must have identical data
         # that is all rows in A must be in B, and vice-versa
-
```

Comment:
Is this just a formatting change?

Reply:
Yes, this is just a formatting change caused by black